tasks/task891_gap_coreference_resolution.json

```diff
@@ -0,0 +1,1232 @@
+{
+    "Contributors": [
+        "Selvaganesh Muthu"
+    ],
+    "Source": [
+        "gap",
+        "https://huggingface.co/datasets/gap"
+    ],
+    "Categories": [
+        "Coreference Resolution"
+    ],
+    "Definition": "Read the passage and find the corresponding person for the given pronoun. The pronoun-offset is the character position of first character of the pronoun from start.",
```

Comment:
Instead of "chracter offset", which is a bit difficult for humans to read/comprehend, how about we indicate the target pronoun within the sentence?  For example, we can indicate it as `.... [he] ...` or  `**he**`. 
What do you think?

Reply:
Yeah, we can indicate it that way. But in certain rare cases, the text itself may contain such symbols. Will that be a problem? Shall I change it?